docs/integrators/observing-squad.md

```diff
@@ -241,3 +241,9 @@ cat logs/mx-chain-.......log
 ```
 
 More detailed commands for installing, building and running an Observing Squad using Docker are described [here](https://github.com/multiversx/mx-chain-observing-squad.git). The images (for the Proxy and for the Observers) are published on [Docker Hub](https://hub.docker.com/u/multiversx).
+
+[comment]: # (mx-context-auto)
+
+## **One click instances in AWS**
```

Comment:
Let's update to "One Click Deploy in AWS" to be consistent with marketing communication . We will also add One Click deploy for Tencet / GCP / DO .

Reply:
Done. 
Let me know when we have the other CPs integrated as well and I'll add chapters for each of them :faiar: